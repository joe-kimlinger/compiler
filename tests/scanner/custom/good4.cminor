// String
"-_=&^%$#@!*()|\\"
"hello \n goodbye"
"\t dsadfsd \b"
" \"hello\" "
" 'this works' "
x: string "I should see this\0I shouldnt see this"
