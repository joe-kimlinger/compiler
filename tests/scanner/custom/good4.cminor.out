COMMENT
STRING_LITERAL -_=&^%$#@!*()|\
STRING_LITERAL hello 
 goodbye
STRING_LITERAL t dsadfsd b
STRING_LITERAL  "hello" 
STRING_LITERAL  'this works' 
IDENTIFIER x
COLON
STRING
STRING_LITERAL I should see this
